content/_es/help/get-started/authentication-options.md

```diff
@@ -37,19 +37,18 @@ Si usted elige esta opción de seguridad, siga estos pasos para descargar e inst
    * Aplicaciones para Windows: [1Password](https://1password.com/) y [OTP Manager](https://www.microsoft.com/en-us/store/p/otp-manager/9nblggh6hngn).
    * Aplicaciones para Mac: [1Password](https://1password.com/) y [OTP Manager](https://itunes.apple.com/us/app/otp-manager/id928941247?mt=12).
    * Extensiones para Chrome: [Authenticator](https://chrome.google.com/webstore/detail/authenticator/bhghoamapcdpbohphigoooaddinpkbai?hl=en).
-3. Abra un nuevo navegador e inicie sesión en su cuenta de login.gov a través de <https://secure.login.gov/>.
+3. Abra un nuevo navegador e inicie sesión en su cuenta de Login.gov a través de <https://secure.login.gov/>.
 4. Seleccione “Activar” junto a “Aplicación de autenticación” y siga las instrucciones para escanear o ingresar un código y vincular así la aplicación de autenticación a su cuenta.\
 {: .number-list}
 
-En adelante, cada vez que inicie sesión en login.gov, podrá utilizar los códigos de acceso de uso único generados por la aplicación.
+En adelante, cada vez que inicie sesión en Login.gov, podrá utilizar los códigos de acceso de uso único generados por la aplicación.
 
 ## Llave de seguridad
 
 Por lo regular, una llave de seguridad es un dispositivo físico externo, como una memoria USB, que se conecta al equipo. La llave se vincula a sus cuentas y solo permitirá acceder a estas una vez que esté conectada y activada. Como la llave de seguridad no depende de su teléfono celular, cuenta con un mejor nivel de protección contra el phishing y elementos de seguridad integrados para prevenir el jaqueo en caso de que sea robada.
+ Login.gov solo permite usar llaves que cumplan con los estándares de [FIDO (Fast Identity Online)](https://fidoalliance.org/). Usted puede agregar tantas llaves de seguridad como desee para proteger su cuenta.
```

Comment:
Looks like losing the line between these two blocks of text changes it from a paragraph to a line break:

Before|After
---|---
![image](https://user-images.githubusercontent.com/1779930/126690723-b603b5f0-5ea0-449a-afb5-fd36c52408a4.png)|![image](https://user-images.githubusercontent.com/1779930/126690698-8513d470-20e1-42d5-84fc-c463aa6628fc.png)

```suggestion

Login.gov solo permite usar llaves que cumplan con los estándares de [FIDO (Fast Identity Online)](https://fidoalliance.org/). Usted puede agregar tantas llaves de seguridad como desee para proteger su cuenta.
```